app/models/customer.py

```diff
@@ -1,4 +1,26 @@
 from app import db
+from datetime import datetime
 
 class Customer(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    __tablename__ = "customers"
+    customer_id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, default=datetime.utcnow())
+    
+    videos_checked_out_count = db.Column(db.Integer, default = 0, nullable = True)
+    rentals = db.relationship('Rental',backref='customers', lazy=True)
```

Comment:
Since a rental typically has 1 customer:

```suggestion
    rentals = db.relationship('Rental',backref='customer', lazy=True)
```